pkg/storagemigration/failcleanup.go

```diff
@@ -36,5 +31,23 @@ func failCleanup(root string) error {
 		return err
 	}
 
+	err = SwitchAllContainersStorageDriver(root, "aufs")
```

Comment:
If any of the `removeDirIfExists()` calls above fails, we'd return before reaching this call here, and I believe this could leave the device in an inconsistent state. (Specifically: if any containers were successfully migrated, they'd be tagged as being overlay2 containers, but the corresponding overlay2 data might have been deleted.)